aexpect/client.py

```diff
@@ -205,7 +205,7 @@ def __init__(self, command=None, a_id=None, auto_close=False, echo=False,
             for reader, filename in self.reader_filenames.items():
                 self.reader_fds[reader] = os.open(filename, os.O_RDONLY)
         except (AssertionError, OSError):
-            pass
+            LOG.warning("Failed to open reader '%s'", filename)
```

Comment:
AFAICT this warning message would be appropriate for lines 205-206, but not for 204.  If that's the case, I'd suggest coming up with either a generic message or two different expect blocks that can give a more precise warning (preferred).

Reply:
good catch